test/test_report_helper.rb

```diff
@@ -1,7 +1,16 @@
 require 'minitest/reporters'
 
-junit_reporter = Minitest::Reporters::JUnitReporter.new('jenkins/reports/unit/')
-meantime_reporter = Minitest::Reporters::MeanTimeReporter.new(previous_runs_filename: Rails.root.join('tmp', 'foreman_minitest_reporters_previous_run'),
-                                                              report_filename: Rails.root.join('tmp', 'foreman_minitest_reporters_report'))
+if ENV['GITHUB_ACTIONS'] == 'true'
+  require 'minitest_reporters_github'
+  reporters = [MinitestReportersGithub.new]
+else
+  reporters = [
+    Minitest::Reporters::JUnitReporter.new('jenkins/reports/unit/'),
+    Minitest::Reporters::MeanTimeReporter.new(
```

Comment:
After thinking more on this. Any reason not to add `MeanTimeReporter` in GHA too? While it doesn't give change-over-time stats (as there is no previous run), it gives a nice summary at the end of the run which tests are costing the most time, and I think we should keep that.

Reply:
I was thinking about a related topic: how do we know how many tests were executed. Just because it's green, doesn't mean it ran any tests for example.

As for mean time reporter, would you store an artifact with the previous data that's updated after merge?

Reply:
> I was thinking about a related topic: how do we know how many tests were executed. Just because it's green, doesn't mean it ran any tests for example.

Yeah, sadly GH doesn't have a way to report this as a text-line from GH, like we have with Jenkins.
We could post a summary (https://github.blog/2022-05-09-supercharging-github-actions-with-job-summaries/ - requires a click) or a comment (no click, but ugly)…

> As for mean time reporter, would you store an artifact with the previous data that's updated after merge?

Not at the moment, no. Or rather: storing the result is easy, but then we also need to retrieve it…